Tests/test_slice.py

```diff
@@ -2663,46 +2672,46 @@ def __delslice__(self, i, j, value):
             def __delitem__(self, index):
                 global setVal
                 setVal = index
-        
+
         del foo()[::]
         self.assertEqual(setVal, slice(None, None, None))
         del foo()[::None]
         self.assertEqual(setVal, slice(None, None, None))
-    
+
     def test_getslice_missing_values(self):
         # missing values are different from passing None explicitly
         class myint(int): pass
         class mylong(long): pass
-        
+
         class foo:
-            def __getslice__(self, i, j):
+            def __getslice__(self, i, j):  # shouldn't be called
                 return (i, j)
```

Comment:
If it shouldn't be called, maybe raise an exception? I'm also fine with just removing `__getslice__`.

Reply:
Removed `__getslice__`